pkg/kubelet/dockertools/kube_docker_client.go

```diff
@@ -126,6 +129,11 @@ func (d *kubeDockerClient) InspectContainer(id string) (*dockertypes.ContainerJS
 }
 
 func (d *kubeDockerClient) CreateContainer(opts dockertypes.ContainerCreateConfig) (*dockertypes.ContainerCreateResponse, error) {
+	// TODO: docker engine-api does not properly respect ShmSize as omitempty
```

Comment:
With the combination of engine-api and Docker other than what we currently ship in Fedora 23 (1.9.1), we don't need omitempty. Sending 0 is fine and will result in Docker applying its default of 64k.